sourcegraph/sourcegraph
# own: repo and directory level drilldown pages for owners and signals

Closes [50799](https://github.com/sourcegraph/sourcegraph/issues/50799)
Adds repo and directory level drilldown for owners and ownership signals.

## Test plan

1. Navigate to a repository and click on "ownership" -> should navigate to the ownership page for the repo root

![CleanShot 2023-06-01 at 16 19 35@2x](https://github.com/sourcegraph/sourcegraph/assets/5090588/45191786-3b14-4aed-90b5-284d274a5925)

2. Click on "show more" from the ownership box from a directory view -> should navigate to the ownership page for that directory path
![CleanShot 2023-06-01 at 16 18 59@2x](https://github.com/sourcegraph/sourcegraph/assets/5090588/c4a2c474-e766-4abb-a3f7-776312d29e8e)

4. Click on edit ownership -> should navigate to the edit ownership page for the same repository
![CleanShot 2023-06-01 at 16 20 58@2x](https://github.com/sourcegraph/sourcegraph/assets/5090588/0b50d499-ee88-43e9-9721-64cfc283be56)
